pkg/systemd/services.jsx

```diff
@@ -597,8 +597,8 @@ class ServicesPageBody extends React.Component {
         if (failed.size > 0) {
             page_status.set_own({
                 type: "error",
-                title: cockpit.format(cockpit.ngettext("$0 service has failed",
-                                                       "$0 services have failed",
+                title: cockpit.format(cockpit.ngettext(_("$0 service has failed"),
+                                                       _("$0 services have failed"),
```

Comment:
This is wrong -- `_` is an alias for `cockpit.gettext()`, but this is already called inside of `cockpit.ngettext()`.

Reply:
ah right